assets/src/scripts/components/graph/state/layout.js

```diff
@@ -45,7 +49,7 @@ export const resetViewport = function (id) {
  * @type {Object}
  */
 export const getViewport = memoize(opts => createSelector(
-    state => state[MOUNT_POINT].viewports || {},
+    (state) => getNamespace(state).viewports || {},
```

Comment:
The parenthesis around state on the left side of the arrow are not needed. Consider eliminating them (not sure but Codacy may flag these).

Reply:
ok

Reply:
I find these parentheses all over the place with no standard pattern. Would you like a story to clean some of this in all locations?

Reply:
Sometimes you do need the parenthesis with arrow functions. Particularly if it is not a simple return. I don't think cleaning this up at this point would be helpful.